static/app/utils/replays/replayReader.tsx

```diff
@@ -83,14 +101,27 @@ export default class ReplayReader {
     // Find LCP spans that have a valid replay node id, this will be used to
     const highlights = createHighlightEvents(spans);
 
-    // Create a final event so that rrweb has the same internal duration as the
+    // Init bookend event so that rrweb has the same internal duration as the
     // root event.
-    // This allows the scrubber to view timestamps after the last captured
-    // rrweb event, like if a memory span was captured after the last mouse-move,
-    // but before page unload.
+    // This allows the scrubber to view timestamps before and after the last
+    // captured rrweb event, like if a memory span was captured after the last
+    // mouse-move, but before page unload.
+    const {startTimestsampMS, endTimestampMS} = this._getTimestampsMS();
+
+    const sortedRRwebEvents = this._rrwebEvents.sort((a, b) => a.timestamp - b.timestamp);
```

Comment:
We call `mergeAndSort....` before returning, is the sorting there still needed?

Reply:
I think the followup sort is needed too rightnow. That call is after we've inserted the highlights, which are going to be inserted into the middle of the list.